core/src/components/range/range.ios.vars.scss

```diff
@@ -4,76 +4,79 @@
 // --------------------------------------------------
 
 /// @prop - Padding top/bottom of the range
-$range-ios-padding-vertical:                  8px !default;
+$range-ios-padding-vertical: 8px !default;
 
 /// @prop - Padding start/end of the range
-$range-ios-padding-horizontal:                16px !default;
+$range-ios-padding-horizontal: 16px !default;
 
 /// @prop - Height of the range slider
-$range-ios-slider-height:                     42px !default;
+$range-ios-slider-height: 42px !default;
 
 /// @prop - Width of the area that will select the range knob
-$range-ios-hit-width:                         42px !default;
+$range-ios-hit-width: 42px !default;
 
 /// @prop - Height of the area that will select the range knob
-$range-ios-hit-height:                        $range-ios-slider-height !default;
+$range-ios-hit-height: $range-ios-slider-height !default;
 
 /// @prop - Height of the range bar
-$range-ios-bar-height:                        2px !default;
+$range-ios-bar-height: 4px !default;
 
 /// @prop - Background of the range bar
-$range-ios-bar-background-color:              rgba(var(--ion-text-color-rgb, 0, 0, 0), .1) !default;
+$range-ios-bar-background-color: #e4e4e6 !default;
```

Comment:
I believe we used `--ion-text-color-rgb` so that background could be adjusted globally/adjusted for dark mode. My understanding is this fixes a bug where the background was no solid. Maybe a step color is more appropriate here?